pytest_tests/resources/load_params.py

```diff
@@ -1,27 +1,29 @@
 import os
 
 # Load node parameters
-LOAD_NODES = os.getenv("LOAD_NODES", "").split(",")
-LOAD_NODE_SSH_USER = os.getenv("LOAD_NODE_SSH_USER", "root")
-LOAD_NODE_SSH_PRIVATE_KEY_PATH = os.getenv("LOAD_NODE_SSH_PRIVATE_KEY_PATH")
+LOAD_NODES = os.getenv("LOAD_NODES", "127.0.0.1").split(",")
+LOAD_NODE_SSH_USER = os.getenv("LOAD_NODE_SSH_USER", "k6")
+LOAD_NODE_SSH_PRIVATE_KEY_PATH = os.getenv(
+    "LOAD_NODE_SSH_PRIVATE_KEY_PATH", "../neofs-dev-env/services/k6_node/id_ed25519"
+)
 BACKGROUND_WRITERS_COUNT = os.getenv("BACKGROUND_WRITERS_COUNT", 10)
 BACKGROUND_READERS_COUNT = os.getenv("BACKGROUND_READERS_COUNT", 10)
 BACKGROUND_OBJ_SIZE = os.getenv("BACKGROUND_OBJ_SIZE", 1024)
 BACKGROUND_LOAD_MAX_TIME = os.getenv("BACKGROUND_LOAD_MAX_TIME", 600)
 
 # Load run parameters
 
-OBJ_SIZE = os.getenv("OBJ_SIZE", "1000").split(",")
-CONTAINERS_COUNT = os.getenv("CONTAINERS_COUNT", "1").split(",")
+OBJ_SIZE = [int(o) for o in os.getenv("OBJ_SIZE", "1000").split(",")]
+CONTAINERS_COUNT = [int(c) for c in os.getenv("CONTAINERS_COUNT", "1").split(",")]
```

Comment:
Why a loop here? I'd expect these to be simple numbers.

Reply:
It is a little bit strange design, but as I understand, the original intention was to pass test parametrization variables through env vars, but these vars are strings (or more precisely a list of strings, since .split() was used). But these variables are used as ints inside the test, so we need to convert them to ints here. Otherwise inside the test there are numerous TypeError exceptions.